WalletWasabi.Tests/UnitTests/WabiSabi/Backend/CoinVerifierTests.cs

```diff
@@ -208,4 +226,12 @@ private List<Coin> GenerateCoins(int numToGen)
 
 		return coins;
 	}
+
+	private void ScheduleVerifications(CoinVerifier coinVerifier, IEnumerable<Coin> coins, TimeSpan? delay = null)
```

Comment:
I am not sure about what to return here. It can be the Task but sometimes it will be null. Or a bool that it was successful or not. From another perspective this is just scheduling something - it is like a fire and forget. Returning a Task might imply you should wait for it - but that is not the case.